doc/contributions-guide.md

```diff
@@ -115,3 +107,7 @@ are welcome, please use the [netopenconfig Google
 group](mailto:netopenconfig@googlegroups.com), or the [public repository
 issues](https://github.com/openconfig/public/issues). 
 
+[0]: Previously, changes were merged in an upstream repository, but since
```

Comment:
Is it fair to say that contributions should follow semantic versioning or something similar?  Should that be explicitly called out and documented in these guidelines?

Reference: https://semver.org/

Reply:
Looks like this is already detailed here but should there be a pointer to it from the contributors guide?

https://openconfig.net/docs/semver/

Reply:
Added a link. Thanks.